src/pages/Task.tsx

```diff
@@ -66,10 +66,14 @@ export const Task: React.FC = () => {
     annotation,
     latestExecution,
     versionMetadata,
+    patch,
```

Comment:
Sounds good! It won't work locally because I think the requester fields in the database are incorrect for the one patch on the commit queue, but it should work for deployed version (I'll check once with `yarn prod` once I can find a patch on a commit queue...)